docs/developing.md

```diff
@@ -12,19 +12,28 @@ You can use the script `scripts/setup-env.sh` to quickly set up local monerod-te
 ganache instances. If you need to later kill `ganache`, `monerod`, `monero-wallet-rpc`, or
 `swapd` instances, you can use `scripts/cleanup-test-processes.sh`.
 
-## Deploying or using deployed SwapFactory.sol
-
-The swap program uses a "factory" contract for the Ethereum side to reduce gas costs from deploying a new contract for each swap. The contract can be found in [here](../ethereum/contracts/SwapFactory.sol). For each new swap, the eth-holding party will call `NewSwap` on the factory contract, initiating a swap instance inside the contract.
```

Comment:
Other than "factory" => "creator" substitution, no other content changes were made in this file. Added newlines are causing the larger diff. This was a 340 character long line before.